acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/pageobjects/BitbucketPluginConfiguration.java

```diff
@@ -0,0 +1,59 @@
+package it.com.atlassian.bitbucket.jenkins.internal.pageobjects;
+
+import org.jenkinsci.test.acceptance.po.Control;
+import org.jenkinsci.test.acceptance.po.JenkinsConfig;
+import org.jenkinsci.test.acceptance.po.PageArea;
+import org.jenkinsci.test.acceptance.po.PageAreaImpl;
+import org.openqa.selenium.support.ui.Select;
+
+/**
+ * todo
+ */
+public class BitbucketPluginConfiguration extends PageAreaImpl {
```

Comment:
That's not the naming convention the framework uses, and I think the package name should be enough to differentiate (plus the two aren't likely to co-exist, because of [the incompatibility of the acceptance-test-harness framework and the core Jenkins dependencies](https://github.com/jenkinsci/acceptance-test-harness/blob/master/docs/EXTERNAL.md)).